syntax = "proto3";

//pdc
message PB_UssSector_FrameHeader{
    uint32 nFrameId = 1;
    uint32 nRervered = 2;
    uint64 ITimeMsec = 3;
}

message PB_UssSectorInfo
{
    repeated uint32 PDC_Distance = 1;
    repeated uint32 LAEB_Distance = 2;
    repeated uint32  SensorStatus = 3;
    uint32 SystemStatus = 4;
    uint32 nRervered = 5;
}

message PB_UssSectorOutputData
{
    PB_UssSector_FrameHeader  FrameHead = 1;
    PB_UssSectorInfo SectorData = 2;
}